rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,589 @@
+      maxLength: 80
+      pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
+      example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    ErrorMessage:
+      description: Error message information.
+      type: object
+      required:
+        - timestamp
+        - path
+        - status
+        - error
+        - message
+      properties:
+        timestamp:
+          type: string
+          format: date-time
+          description: The server time when the error was generated.
+          example: 2019-05-09T14:47:13.101+0000
+        path:
+          type: string
+          description: The path for the REST request that generated this error.
+          example: /validation/validate
+        status:
+          type: integer
+          format: int32
+          description: The corresponding HTTP status code.
+          example: 500
+        code:
+          type: integer
+          format: int32
+          description: >
+            An optional more specific code corresponding to the error (specific codes need to be
+            documented).
+          example: 0
+        error:
+          type: string
+          description: The corresponding HTTP reason phrase.
+          minLength: 1
+          example: Bad Request
+        message:
+          type: string
+          description: Specific information about the error.
+          minLength: 1
+          example: Security markings validation failed
+        details:
+          type: array
+          items:
+            type: string
+          description: Optional detailed information about the error.
+          example: [
+            No security markings provided,
+            Unexpected field 'Soccer'
+          ]
+    ReasonForInvalidation:
+      description: High level explanation of why the provided markings were deemed invalid.
+      type: string
+      example: Given classification is above the system high classification
+    SecurityMarkings:
+      description: Security markings.
+      type: object
+      properties:
+        ISM:
+          $ref: '#/components/schemas/ISM'
+#        BannerMarkings:
+#          $ref: '#/components/schemas/BannerMarkings'
+    InvalidSecurityMarkings:
+      description: Security markings found to be invalid and reason for invalidation.
+      type: object
+      properties:
+        invalidISM:
+            $ref: '#/components/schemas/InvalidISM'
+#        invalidBannerMarkings:
+#            $ref: '#/components/schemas/InvalidBannerMarkings'
+#    BannerMarkings:
+#      description: Banner markings string, delimited by '//'
+#      type: string
+#      example: U//FOUO
+#    InvalidBannerMarkings:
+#      description: Banner markings string, delimited by '//', and reson for invalidation
+#      type: object
+#      properties:
+#        bannerMarkings:
+#          $ref: '#/components/schemas/BannerMarkings'
+#        reasonForInvalidation:
+#          $ref: '#/components/schemas/ReasonForInvalidation'

```

Comment:
❓ Same comment as above. Why was this commented? If this is going to be uncommented this could be a good place to have a TODO.

Reply:
Same answer as above; very certain this will be needed in the future. Adding TODO 👍